src/it/resources/org/checkstyle/suppressionxpathfilter/genericwhitespace/InputXpathGenericWhitespaceNestedGenericsThree.java

```diff
@@ -2,7 +2,7 @@
 
 import java.io.Serializable;
 
-public class SuppressionXpathRegressionGenericWhitespaceNestedGenericsThree {
+public class InputXpathGenericWhitespaceNestedGenericsThree {
```

Comment:
Some of these have `Generics`, others have `Generic` (not related to check name).

Just remove the Generic(s) words completely.

Reply:
Done and rebased on master.
I couldn't come up with good suffixes here, so left the numbering as is

Reply:
That is fine.